src/OhadSoft.AzureLetsEncrypt.Renewal/Util/CertificateHelper.cs

```diff
@@ -50,7 +50,8 @@ public static async Task<IReadOnlyList<string>> GetLetsEncryptHostNames(IAzureWe
                 Trace.TraceInformation("Reading ARM certificate query response");
                 var body = await response.EnsureSuccessStatusCode().Content.ReadAsStringAsync().ConfigureAwait(false);
 
-                var letsEncryptCerts = ExtractCertificates(body).Where(s => s.Issuer.Contains(staging ? "Fake LE" : "Let's Encrypt"));
+                var issuerNames = staging ? new[] { "Fake LE" } : new[] { "Let's Encrypt", "R3", "R4", "E1", "E2" };
+                var letsEncryptCerts = ExtractCertificates(body).Where(s => issuerNames.Any(i => s.Issuer.StartsWith(i, StringComparison.InvariantCulture)));
```

Comment:
I think ordinal would be better than invariant culture.
Also, I'm not sure about the spec, but if it's case insensitive then use `StringComparison.OrdinalIgnoreCase`